RIP-3 Nft Bank/RIP-3 Nft Bank.md

```diff
@@ -0,0 +1,81 @@
+---
+rip: 3
+title: Nft Bank
+author:  @butterfiolee
+discussions-to: <URL>
+status: Draft
+created: 2022-03-21
+---
+
+## Abstract
+This is a way for users to safely store their nft's outside of their wallet.
+
+## Motivation
+Players over time will accumulate a large amount of assets through the collection of gaming assets as well as other general non-fungible tokens such as art. Wallets as they were originally conceived weren't meant to handle such a large volume of items and eventually they become cumbersome to organize.
```

Comment:
Wallets are meant to hold large volumes of asset, but existing user interfaces are not designed for the organization of large amounts of assets. Can you change this sentence a bit?